pyodide-build/pyodide_build/out_of_tree/utils.py

```diff
@@ -30,7 +30,7 @@ def initialize_pyodide_root(*, quiet: bool = False) -> None:
     if "PYODIDE_ROOT" in os.environ:
         return
     try:
-        os.environ["PYODIDE_ROOT"] = str(search_pyodide_root(__file__))
+        os.environ["PYODIDE_ROOT"] = str(search_pyodide_root(Path.cwd()))
```

Comment:
Is this a safe change? It does something pretty different. Could you explain the logic before vs now?

Reply:
I think there was a bug in the logic before, which caused an error here: https://github.com/pyodide/pyodide/discussions/3669.

Previously it was finding the pyodide root from the parents of that file. So if we don't install `pyodide-build` in a virtual environment inside the project folder, it failed.